### PR TITLE
improve int test dev experience

### PR DESCRIPTION
**Pull request by jroebu14**
**Overall change:**
Improves the DX when working with integration tests by offering useful commands and hiding the server logs from the console.

```
npm run test:integration
```

To run tests for a single page type with the watch task:

```
npm run test:integration -- --pageTypes=articles --watch
```

To run tests for multiple page types:

```
npm run test:integration -- --pageTypes=articles,frontPage
```

To run tests for a single page type with the watch task and webpack hot reloading of application code:

```
npm run test:integration -- --pageTypes=liveRadio --watch --dev
```

To stop running tests immediately when there is a failure - NB this is useful when you want to reduce noise if there are a lot of failing tests and you want to inspect one failing test at a time:

```
npm run test:integration -- --bail
```

Any other Jest CLI args and flags can be passed along in the `test:integration` script.

Screenshot 2020-06-05 at 09 05 15

_the command for this screenshot was_ `npm run test:integration -- --pageTypes=photoGalleryPage --dev --watch`

**Code changes:**

- Added node script `src/integration/utils/runTests/index.js` to run integration tests.
- Updated integration test scripts in `package.json`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by karinathomasbbc — APPROVED** [review]
Nice, this will make running these tests sooo much easier

**Review by rhenshaw56 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -34,8 +34,8 @@
     "test:puppeteer": "jest --ci --env=jsdom --colors ./puppeteer",
     "test:unit": "test -z $CI && jest --env=jsdom --coverage --colors --testPathIgnorePatterns=\"src/integration\" \"puppeteer\" || jest --ci --runInBand --env=jsdom --coverage --colors --testPathIgnorePatterns=\"src/integration\" \"puppeteer\"",
     "test:unit:watch": "npm run test:unit -- --watch",
-    "test:integration": "npm run stop && npm run build:local && (npm run start & wait-on -t 20000 http://localhost:7080/status) && (jest --runInBand --colors ./src/integration/pages/**/*) ; npm run stop",
-    "test:integration:ci": "jest --ci --runInBand --colors ./src/integration/pages/**/*",
+    "test:integration": "node src/integration/utils/runTests/index.js",
+    "test:integration:ci": "JEST_SILENT_REPORTER_DOTS=true npm run test:integration -- --ci --reporters=jest-silent-reporter",
```
Comment (DenisHdz) [inline package.json:38]: Can this env variable `JEST_SILENT_REPORTER_DOTS=true` and this `reporters=jest-silent-reporter` flag also be checked within the `runTests/index.js` as you do with `ci`?

**Review by DenisHdz — APPROVED** [review]
Excellent work 💯, just a question:

File: package.json (hunk 1 above)
Comment (jroebu14) [reply package.json:38]: I suppose it could be but I'm not sure I see the benefit of doing that as it would add more logic to `runTests/index.js`

**Pull request merged**